### PR TITLE
[13.0] EDI refactor ack handling w/ its own type

### PR DESCRIPTION
**Pull request by simahawk**
NOTE: edi_storage ack handling has been put on hold because we still have to implement automatic process of incoming (eg: receive/retrieve files from outside).

### CODE REVIEW

**Comment by simahawk** [comment]
@etobella  rebased and ready for review

File: edi/views/edi_exchange_record_views.xml [hunk 1]
```diff
@@ -53,12 +53,13 @@
                             <field name="exchange_filename" invisible="1" />
                             <field name="direction" invisible="1" />
                         </group>
-                        <group
-                            name="ack"
-                            attrs="{'invisible': [('ack_needed', '=', False)]}"
-                        >
+                        <!-- TODO: move to a page? -->
```
Comment (etobella) [inline edi/views/edi_exchange_record_views.xml:56]: Yes, probably better there, maybe invisible if the type has not an ack_type_id defined

Comment (simahawk) [reply edi/views/edi_exchange_record_views.xml:56]: I'd leave the TODO there for now. We can change it later.
What about the rest? Approve-able? :)

**Review by etobella — APPROVED** [review]
Yes, it is, just was a comment, and can be managed later, jejeje

**Pull request merged**